libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt

```diff
@@ -60,6 +60,16 @@ class FeedbackEvent {
          */
         const val FEEDBACK_TYPE_INACCURATE_GPS = "inaccurate_gps"
 
+        /**
+         * Feedback type *incorrect visual*: wrong visual of navigation route
+         */
+        const val FEEDBACK_TYPE_INCORRECT_VISUAL = "incorrect_visual"
```

Comment:
The value has changed to `incorrect_visual_guidance` and `incorrect_audio_guidance`

Reply:
will update it.